### DISCUSSION TITLE
How to invalidate cache?

### DISCUSSION THREAD

**Discussion by geosigno** [apollographql/apollo-server#5361] [Q&A]
I've been through the entire documentation to chose which caching solution I should use. So far i think automatic persisted query would be the best (and easiest) way to implement it.

However, i cannot find anything about invalidate those query? My flow is quite basic, I need to cache my _getMessage_ query, but I also want to clear the cache on that query as soon as the mutation createMessage is called. Is there no simple way to do that?

Thanks!

**Comment by glasser** [comment]
apollo-server-plugin-response-cache doesn't support invalidation. It's certainly a requested feature (see #3228). This plugin (forked from our plugin) supports invalidation: https://github.com/thematters/apollo-response-cache
